### ISSUE TITLE
Missing R script

### ISSUE THREAD

**Issue by scholer** [fdannenberg/dna#1]
Hi,
Congratulations on your Nature publication and the nice manuscript on arXiv. And thank you for releasing this software for the benefit of the DNA nanotech community!

I discovered that there seems to be a R script missing from the repository. After the simulation completes, I can see in the source code that it tries to execute `RGraphs/scatterplot.R` using `CommandlineCaller.call(command)`. However, this file `RGraphs/scatterplot.R` is not present in the repository..

To be specific, the reference to the R script is in `src/writers/RCommand.java, line 65`. There is also a reference to `RGraphs/jon-graph.R` in line 44 of the same file, although I don't think this is ever called explicitly.

Thanks again!

**Comment by fdannenberg** [comment]
Dear Rasmus,

This is indeed a faulty behaviour: the scatterplot is depreciated, but 
the calculated temperatures are still used. So just removing the call to 
the R script will fix this.

If you pull from master, this should now be fixed. The call to jon-graph 
is also gone.

Thanks for checking out the software, let me know if anything else pops 
up. I have to admit that in my environment, calls to R just fail 
quietly, sorry about that.

Best wishes,

Frits

**Comment by scholer** [comment]
Hi Frits,
Thanks for responding to this issue, glad to know it wasn't actually an issue at all. I pulled your updates and the simulation and analysis still runs smoothly. I've made a few changes to make it compatible with Windows, which doesn't like colons (":") in filenames. 

May I ask, are you still adding new features to this software package? I have two things I would really like to use this for. One is making an animated visualization of the DNA strands as they assemble. This would mostly be for teaching purposes, using small structures and a short timeframe. I'm hoping that a simulation would provide a more realistic picture of the stochastic and reversible nature of how DNA nanostructures assemble. Right now, many students see Shawn's [DNA Origami folding animation](https://www.youtube.com/watch?v=p4C_aFlyhfI) and think that's how it actually assembles.

The second feature I would like to have is a less restrictive assembly model, which does not rely on a set of predefined legal states. We are sometimes seeing our assemblies producing side-products we hadn't predicted. It would be great if we could predict these before we spend money ordering the oligos and time characterizing the assemblies. Obviously, we can't properly define the set of legal states when we don't know in advance what side-products we are going to get. 

I was thinking of maybe trying to implement some of this as a project on the side. But obviously if you are already working on these things, I will not duplicate the work effort.

Thanks again,
  Rasmus
